airflow/providers/jenkins/hooks/jenkins.py

```diff
@@ -46,3 +47,24 @@ def __init__(self, conn_id: str = default_conn_name) -> None:
     def get_jenkins_server(self) -> jenkins.Jenkins:
         """Get jenkins server"""
         return self.jenkins_server
+
+    def get_build_building_state(self, job_name: str, build_number: int) -> bool:
```

Comment:
can you add unit test to the hook function?

Reply:
definitely, I am adding one shortly.